src/api/loaders/csv.js

```diff
@@ -0,0 +1,9 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+
+ezs.use(ezsBasics);
+
+export default config => stream =>
+  stream
```

Comment:
Better but you should configure eslint for your IDE. This project has a specific eslint configuration with tabs made of 4 spaces :)